R/post_ts.R

```diff
@@ -54,10 +54,8 @@ post_ts = function(files, on_exists=c("stop", "skip", "replace", "merge","archiv
     ts_id <- locate_ts(var_src=ts_path$var_src, site_name=ts_path$site_name, by="either")
     if (is.na(ts_id)) {
       ts_id <- item_create(site_root, title=ts_path$ts_name)$id
+      repair_ts(ts_path$var_src, ts_path$site_name)
```

Comment:
oh, right. i moved all of the tagging (`scheme`, `key`, `type`) into `repair_ts`, so it makes sense that it would need to be called at least once.